src/core/Akka.Remote/Routing/RemoteRouterConfig.cs

```diff
@@ -20,114 +20,87 @@ namespace Akka.Remote.Routing
     /// routees on defined target nodes. Delegates other duties to the local <see cref="Pool"/>,
     /// which makes it possible to mix this with built-in routers such as <see cref="RoundRobinGroup"/> or custom routers.
     /// </summary>
-    public sealed class RemoteRouterConfig : Pool 
+    public sealed class RemoteRouterConfig : Pool, IEquatable<RouterConfig>
     {
-        public class RemoteRouterConfigSurrogate : ISurrogate
```

Comment:
We may need to add this back in order to maintain serialization support for people who are remotely deploying routers.